src/main/groovy/app/BitBucketDocumentTemplatesStore.groovy

```diff
@@ -40,9 +41,10 @@ class BitBucketDocumentTemplatesStore implements DocumentTemplatesStore {
 
         Feign.Builder builder = Feign.builder()
 
-        if (System.getenv("BITBUCKET_USERNAME") && System.getenv("BITBUCKET_PASSWORD")) {
+        def bitbucketUserName = System.getenv("BITBUCKET_USERNAME")
+        if (bitbucketUserName && System.getenv("BITBUCKET_PASSWORD")) {
```

Comment:
@clemensutschig why not assign the password also to a dedicated variable as you do with the username?

Reply:
hey @metmajer  - I left this as it was, but as I anyway want to introduce proxy lookup - I'll fix this as well